zcash_client_backend/src/data_api.rs

```diff
@@ -560,6 +560,9 @@ pub trait WalletRead {
         account: AccountId,
         max_height: BlockHeight,
     ) -> Result<HashMap<TransparentAddress, Amount>, Self::Error>;
+
+    /// Returns a vector with the IDs of all accounts known to this wallet.
+    fn get_account_ids(&self) -> Result<Vec<AccountId>, Self::Error>;
```

Comment:
Document this change to the `WalletRead` trait in the changelog (add it to the existing list of changes there in the "Unreleased" section).

Reply:
Done

Reply:
I also added a small `.editorconfig` file to preserve the space style indentation in that file, since my editor defaulted to tabs there.